python/tank/bootstrap/manager.py

```diff
@@ -120,14 +120,37 @@ def __repr__(self):
 
         repr  = "<TkManager "
         repr += " User %s\n" % self._sg_user
-        repr += " Cache fallback path %s\n" % self._bundle_cache_fallback_paths
+        repr += " Cache fallback path %s\n" % self._get_bundle_cache_fallback_paths()
         repr += " Caching policy %s\n" % self._caching_policy
         repr += " Plugin id %s\n" % self._plugin_id
         repr += " Config %s %s\n" % (identifier_type, self._pipeline_configuration_identifier)
         repr += " Allows config overrides %s\n" % self._allow_config_overrides
         repr += " Base %s >" % self._base_config_descriptor
         return repr
 
+    def _get_bundle_cache_fallback_paths(self):
+        """
+        Retuns a list containing both the user specified bundle caches and the ones specified by
+        Toolkit.
```

Comment:
Just illustrate stuff like order, can we add some examples here showing what it looks like in practice with/without  the bundle_cache_fallback_paths bootstrap property and the special env vars set?